buzzmobile/launch/includes/sim_spawn_car.sh

```diff
@@ -1,5 +1,5 @@
 #! /bin/bash
 
-sleep 3
+sleep 15
```

Comment:
Its a good kludge, there should at least be a todo to investigate what that actually relies on, and to instrument something that manages process deps better.